charts/budibase/templates/minio-service-deployment.yaml

```diff
@@ -72,6 +72,9 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
     {{- end }}
+      {{- with .Values.schedulerName }}
```

Comment:
I think using `with` here might not be the correct syntax. From having a look at: https://helm.sh/docs/chart_template_guide/control_structures/#modifying-scope-using-with

I see that the `with` will rewrite `.` to mean the value of `.Values.schedulerName`. In this case that would mean the inner usage of `.Values.schedulerName` wouldn't exist. 

Two options
- We can switch the usages in this pr over to `if` instead
- Update the inner `.Values.schedulerName` in each case to just be `.`

Reply:
`with` is not required in these cases. So changed it to `if`. It should be good now.